File: Robot/Wrapper.swift

/**
 * A simple Wrapper class around any value
 */
public class Wrapper<T> {
	
	private var value: T
	
	public init(_ value: T) {
		self.value = value;
	}
	
	public func getValue() -> T {
		return value;
	}
	public func setValue(_ newValue: T) {
		value = newValue;
	}
}

File: Robot/Java/Helper/Wrapper.java
package Robot.Java.Helper;

/**
 * A simple Wrapper class around any value
 */
public class Wrapper<T> {
	
	private T value;
	
	public Wrapper(T value) {
		this.value = value;
	}
	
	public T getValue() {
		return value;
	}
	public void setValue(T newValue) {
		value = newValue;
	}
}
